extension/chrome/elements/compose-modules/compose-render-module.ts

```diff
@@ -98,9 +98,11 @@ export class ComposeRenderModule extends ViewModule<ComposeView> {
       const thread = await this.view.emailProvider.threadGet(this.view.threadId, 'metadata');
       const inReplyToMessage = thread.messages?.find(message => message.id === this.view.replyMsgId);
       if (inReplyToMessage) {
-        this.view.replyParams.inReplyTo = inReplyToMessage.payload?.headers?.find(
-          header => header.name === 'Message-Id' || header.name === 'Message-ID'
-        )?.value;
+        const msgId = inReplyToMessage.payload?.headers?.find(header => header.name === 'Message-Id' || header.name === 'Message-ID')?.value;
+        if (msgId) {
+          this.view.sendBtnModule.additionalMsgHeaders['In-Reply-To'] = msgId;
+          this.view.sendBtnModule.additionalMsgHeaders.References = msgId;
```

Comment:
`references` header should include value of previous msg + `msgId`, like at https://github.com/FlowCrypt/flowcrypt-browser/blob/aa27485fa1229beeb255b426d6ec9ef20c807b73/extension/chrome/elements/compose-modules/compose-render-module.ts#L121

Reply:
Thanks, I've fixed this as well.